base/loading.jl

```diff
@@ -415,14 +415,15 @@ end
 evalfile(path::AbstractString, args::Vector) = evalfile(path, UTF8String[args...])
 
 function create_expr_cache(input::AbstractString, output::AbstractString)
-    isfile(output) && rm(output)
```

Comment:
no, this undoes https://github.com/JuliaLang/julia/pull/12723 - if precompilation fails then any existing ji file should also be removed